src/components/FormLabel/FormLabel.tsx

```diff
@@ -16,6 +16,7 @@ export interface Props {
   id?: string
   label?: any
   helpText?: any
+  inline?: boolean
```

Comment:
I think we can probably make this always present, but have a `defaultProps: { inline: false }` :)

@tinkertrain What you thinky?

Reply:
Update! Another convention we've used for a while now is to prefix boolean based props with `is`. So it would be `isInline`. I think it's a little clearer to the component consumer, as a boolean value is implied.

Unfortunately, since it's a newer(ish) convention, it's consistent through the codebase :(. Something I'm working on cleaning up overtime.

Reply:
Makes sense to me! I like the convention